frontend/src/metabase/entities/actions/actions.ts

```diff
@@ -201,6 +206,50 @@ const Actions = createEntity({
   actions: {
     enableImplicitActionsForModel,
   },
+  objectActions: {
+    createPublicLink: createAction(
+      CREATE_PUBLIC_LINK,
+      ({ id }: { id: number }) => {
```

Comment:
Can we use `WritebackActionId` from `metabase-types/api` here?

Reply:
Yeah, I could, but I wonder why we need to export every entity id type. And more confusingly some of them are `string` while some are `number`. In an ideal world, I'd expect every entity ID type to be the same e.g. string.